GNUmakefile

```diff
@@ -296,7 +296,7 @@ test-nomad: # dev ## Run Nomad unit tests
 	@echo "==> with packages $(GOTEST_PKGS)"
 	gotestsum --format=testname --rerun-fails=3 --packages="$(GOTEST_PKGS)" -- \
 		-cover \
-		-timeout=20m \
+		-timeout=25m \
```

Comment:
I would say "let's go 30" but we can always up it later.

we will need to be mindful of the oveall actions timeout, too
https://github.com/hashicorp/nomad/blob/3ed9119/.github/workflows/test-core.yaml#L107

```yaml
  tests-groups:
    needs: [checks]
    timeout-minutes: 30
```